samples/test-runner/build.gradle.kts

```diff
@@ -84,6 +93,23 @@ instrumentation {
                             }
                         }
                     }
+                    register("api30") {
+                        deviceName = "API30"
+
+                        scheduling {
+                            quota {
+                                retryCount = 1
+                                minimumSuccessCount = 1
+                            }
+
+                            testsCountBasedReservation {
+                                device = emulator30
+                                maximum = 1
+                                minimum = 1
+                                testsPerEmulator = 1
+                            }
+                        }
+                    }
```

Comment:
Added to sample app to check basic integration.
Can remove after we finish update to 30 API.